isushibsiteaccesssync.module

```diff
@@ -0,0 +1,218 @@
+<?php
+
+/**
+ * @file
+ * Syncs the access list with Active Directory group(s).
+ */
+
+
+/**
+ * Implements hook_menu().
+ */
+function isushibsiteaccesssync_menu() {
+  $items['admin/config/people/isushibsiteaccesssync'] = array(
+    'title' => 'Configure AD Sync',
+    'description' => 'Edit settings for Active Directory sync.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('isushibsiteaccesssync_settings'),
+    'access arguments' => array('administer shibboleth site access ad sync'),
+    'file' => 'isushibsiteaccesssync.admin.inc',
+  );
+
+  return $items;
+}
+
+
+/**
+ * Implements hook_cron().
+ */
+function isushibsiteaccesssync_cron() {
+  // Only run this once per day
+  if (variable_get('isushibsiteaccesssync_day_of_year', 0) == date('z') + 1) {
```

Comment:
How does this work on January 1 when isushibsiteaccesssync_day_of_year = 365?. I would use a variable called isushibsiteaccesssync_last_run = 20151231. If isushibsiteaccesssync_last_run < date('Ymd') then run the sync.